### ISSUE TITLE
Tune Streaming throughput

### ISSUE THREAD

**Issue by balsukrishna** [scylladb/scylla#4884]
I am trying add a node to 6 node multi-AZ cluster. 
There is no read load on the cluster while there is 10-12K write load.

It seemed like streaming is slow! I can get only around 50MB/S while streaming from 3 Nodes. 

Looks like streaming is given less priority in priority manager

https://github.com/scylladb/scylla/blob/080989d29624aeced6fab113ebdcbfa6e85be57b/service/priority_manager.hh#L72

Is there any way to tune streaming throughput?  I am not doing reads at this point of time. So I want to increase streaming speed at the expense of latency

*Installation details*
Scylla version (or git commit hash): 3.0.8-0.20190627.23da53c4f
Cluster size: 6
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS Linux

*Hardware details (for performance issues)*          
AWS : I3en.2xlarge

**Comment by tzach** [comment]
There is no way to control the streaming bandwidth. As long as your system has free resources, the workload prioritization should not slow down streaming.
@asias please advise.

**Comment by asias** [comment]
According @balsukrishna, 

```
I did run scylla_setup. although scylla_ec2_check failed as it is not supported i3en instances, so I skipped that check and ran other setups.
Below are some of the configs generated

cat /etc/scylla.d/io.conf
SEASTAR_IO="--num-io-queues=8 --io-properties-file=/etc/scylla.d/io_properties.yaml"

cat /etc/scylla.d/cpuset.conf
# DO NO EDIT
# This file should be automatically configure by scylla_cpuset_setup
#
# CPUSET="--cpuset 0 --smp 1"
CPUSET="--cpuset 0-7 "
cat /etc/scylla.d/io_properties.yaml
disks:
  - mountpoint: /var/lib/scylla
    read_iops: 186565
    read_bandwidth: 1814927232
    write_iops: 143990
    write_bandwidth: 663844608
```

@glommer Will the failure of scylla_ec2_check skip any tunning in scylla_setup? Does the above config looks reasonable in i3en.2xlarge?

**Comment by glommer** [comment]
configs look reasonable.

**Comment by slivne** [comment]
@balsukrishna can you share the schema do you have MVs / SIs

**Comment by rahul619anand** [comment]
From our experiments, we found that reducing the compaction throughput (i.e. setting **compaction_static_shares: 100** in our case) helps streaming performance. 
**This helps a new node join the production cluster faster by 2x.** 
We understand that reducing the compaction share can affect read performance but since the node in joining state doesn't serve any request, it was not a problem.

Although we are yet to understand, why the streaming performance decreases as the disk usage increases. We know, that Levelled Compaction Strategy can create high write amplification (~13x in some cases), so we did try running **fstrim** on a regular basis, although its effects are very random in nature.
